docker-compose.yml

```diff
@@ -166,7 +166,7 @@ services:
         container_name: plex
         image: plexinc/pms-docker:${PMSTAG}
         restart: unless-stopped
-        network_mode: "host"
+        network_mode: "bridge"
```

Comment:
This is fine - I switch to `bridge` and leave the port mappings - this will probably be the most helpful in the long run.